packages/esm-patient-chart-app/src/clinical-views/encounter-tile/clinical-views-summary.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { memo, useMemo } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { type ConfigObject, useConfig } from '@openmrs/esm-framework';
+import { getEncounterTileColumns } from '../utils';
+import { EncounterTile } from './encounter-tile.component';
+import { type MenuCardProps } from '../types';
+
+interface OverviewListProps {
+  patientUuid: string;
+}
+
+const ClinicalViewsSummary: React.FC<OverviewListProps> = memo(({ patientUuid }) => {
+  const config = useConfig<ConfigObject>();
```

Comment:
There's not really any point to using the `ConfigObject` type here. It's more useful to use an app-specific type, but if there is not, declare the type here appropriately.

```suggestion
  const config = useConfig();
```

Reply:
It also seems like it could be cleaner to write:

```ts
const {
  tileDefinitions,
  trueConceptUuid,
  falseConceptUuid,
  otherConceptUuid
} = useConfig();
```